### PR TITLE
Clarify that reveal.js 3.x is needed

### PR DESCRIPTION
**Pull request by takluyver**
Closes gh-702

### CODE REVIEW

**Comment by damianavila** [comment]
+ 1 as I said in the issue.

**Comment by mpacer** [comment]
Looks good but I still think that the default of using the cdn is a better default for most people unless we want to explicitly help people download an up-to-date version of reveal.js.

**Comment by mpacer** [comment]
Actually can you add a reference to the latest example of reveal.js that works? Also a test to make sure that the version mentioned in the docs works? That way, we’ll know if it ever breaks with the a nbconvert change?

**Comment by takluyver** [comment]
I've added a note to the docs showing how to get reveal 3.6. Adding a test that reveal.js works would be a big job, however - it involves executing Javascript in a browser, which the nbconvert tests are not set up to do. And we know from the notebook tests what a pain it is.

File: docs/source/usage.rst [hunk 1]
```diff
@@ -96,9 +96,18 @@ Reveal.js HTML slideshow
...
+  Reveal.js library (version 3.x) in the same directory where ``your_talk.slides.html`` is
   located, or point to another directory using the ``--reveal-prefix`` alias.
 
+  To get a compatible version of revealjs in the current folder:
+
+  .. code-block:: shell
+
+    git clone https://github.com/hakimel/reveal.js.git
+    cd reveal.js
+    git checkout 3.6.0
```
Comment (damianavila) [inline docs/source/usage.rst]: We are using reveal.js 3.5.0 in the serve post-processor, we should keep it consistent and checkout 3.5.0 until we decide to upgrade the whole thing to 3.6.0, I think.

Comment (takluyver) [reply docs/source/usage.rst]: Thanks Damian, updated.

**Comment by mpacer** [comment]
Closing since we now have 732 merged.

**Pull request closed without merging**